incompatibility-allowlist

```diff
@@ -41,7 +41,21 @@ firefox-accounts.account-ecosystem.*
 
 # Bug 1804256
 # Delete mozillavpn.metrics and mozillavpn.baseline after fixing typo in https://github.com/mozilla/probe-scraper/pull/553
-# mozillavpn was treated as Glean.js app after typo got fixed, which resulted in these 
+# mozillavpn was treated as Glean.js app after typo got fixed, which resulted in these
 # ping schemas being removed (since they don't exist for Glean.js)
 mozillavpn.metrics.*
 mozillavpn.baseline.*
+
+# Bug 1815432
+# Delete unused glean crash pings for glean applications other than firefox-desktop
+firefox-desktop-background-update.crash.*
```

Comment:
this one isn't completely empty, but per discussion with chutten it should still be deleted